spec/spec_helper.rb

```diff
@@ -12,6 +12,7 @@
       VCR.request_matchers.uri_without_param('api-version')
     ]
   }
+  #config.debug_logger = File.open('vcr.debug', 'w')
```

Comment:
is this on purpose?

Reply:
Yes, otherwise I always have to look it up when I need it. :)